remaining packer content (#13285)
hashicorp/packer: website/content/docs/commands/fix.mdx

---
description: |

>   The `packer fix` command updates backward incompatible parts of a template so that you can use it with the latest version of Packer. 

Reviewer: ```suggestion
  The `packer fix` command updates backward incompatible templates for the running version of Packer. 
```

i'm assuming that this is for the running version and not the latest.
